src/main/resources/crafter/engine/server-config.properties

```diff
@@ -242,6 +242,17 @@ crafter.security.login.url=/crafter-security-login
 crafter.security.logout.url=/crafter-security-logout
 # The value for the security token used in header authentication
 crafter.security.header.token=my_secure_token
+# The prefix used for all header names
+crafter.security.header.names.prefix=MELLON_
```

Comment:
@avasquez614 @jdrossl is there a reason for this to continue to be defaulted to `MELLON_`?

Reply:
I think it's for backward compatibility no? I wonder so myself, I think we should remove it if possible.

Reply:
yes it's for backward compatibility, we can change it and document it in the release notes

Reply:
@jdrossl remove the MELLON_ prefix and replace it with CRAFTER_ here and in the docs.
Thanks.

Reply:
@alhambrav please include this change in the release notes.